media/server/gstplayer/source/tasks/generic/AttachSource.cpp

```diff
@@ -140,6 +145,7 @@ class MediaSourceAudioCapsBuilder : public MediaSourceCapsBuilder
         {
             caps = m_gstWrapper->gstCapsNewSimple("audio/x-opus", "channel-mapping-family", G_TYPE_INT, 0, nullptr);
             addSampleRateAndChannelsToCaps(caps);
+            firebolt::rialto::AudioConfig audioConfig = m_attachedAudioSource.getAudioConfig();
```

Comment:
Unused variable audioConfig could be removed

Reply:
removed